com.unity.netcode.gameobjects/Components/NetworkTransform.cs

```diff
@@ -370,30 +372,40 @@ private void OnDestroy()
             ReplNetworkState.OnValueChanged -= OnNetworkStateChanged;
         }
 
+        [ServerRpc(Delivery = RpcDelivery.Reliable)]
```

Comment:
shouldn't it?
```suggestion
        [ServerRpc(Delivery = RpcDelivery.Unreliable)]
```

Reply:
No, definitely not here. Sending the RPC is gated by `isDirty`. So, for an object that stops moving, there might be just one last RPC update sent. If it were to get lost, we'd have an object stuck in an old location.

I'm all for using unreliable message. But when we do, we need a retransmit mechanism, acknowledgements, dependency ordering... Which is what we're aiming for with the snapshot system.

Reply:
I'd be interested to test this with artificial packet loss and buffered interpolation, to see how smooth interpolation can make this.
worst comes to worst, we could have an optional manual set dirty every few seconds?

Reply:
I'd rather take the same reliability hit we take everywhere for now than put band-aids. We can then move forward with Snapshot, moving things into snapshot as we want unreliability + eventual consistency.

Reply:
yeah, I see. okay :)

Reply:
making this Unreliable and force-setting NetVar IsDirty=true every second (or 3, 5 secs) sounds like a good cheap solution to me (Sam's idea). shall we?